wasm/__init__.py

```diff
@@ -196,15 +196,15 @@ def spec_validate_functype(ft):
 
 def spec_validate_tabletype(tt):
     limits, elemtype = tt
-    spec_validate_limit(limits, 2 ** 32)
+    spec_validate_limit(limits, constants.UINT32_CEIL)
```

Comment:
Maybe another off by one, since `spec_validate_limit` uses `>` to compare `k`.

Reply:
Added comment to investigate later.  Updating to `UINT32_MAX` resulted in spec test failures.